agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannel.java

```diff
@@ -81,7 +85,7 @@ public static TelemetryChannel create(
   }
 
   public CompletableResultCode sendRawBytes(ByteBuffer buffer) {
-    return internalSend(Arrays.asList(buffer));
+    return internalSend(Arrays.asList(buffer), null);
```

Comment:
in this case, I think we don't want to store the redirect into `redirectMappings` because we may store the redirect from one ikey, and then pick that up from a different ikey.

one option here would be to use something like "unknown" constant to indicate it's not for a known ikey.

another (maybe better) option would be to configure the RedirectPolicy to *only* use ikey redirects when configuring the telemetry channel